hal/lib/athena/PWM.cpp

```diff
@@ -11,6 +11,27 @@
 #include "PortsInternal.h"
 #include "handles/HandlesInternal.h"
 
+namespace hal {
```

Comment:
Should we make these inline helper functions within DigitalPort?  Or just inline somewhere else?  Seems silly to have them be out-of-line functions.